t/unit/concurrency/test_thread.py

```diff
@@ -0,0 +1,29 @@
+from __future__ import absolute_import, unicode_literals
+
+import operator
+import pytest
+
+from celery.utils.functional import noop
+
+from case import skip
+
+
+@skip.if_pypy()
```

Comment:
Because pypy env does not use the test-ci-default.txt, and the test-ci-base.txt does not include any concurrency model requirements (gevent for example).
But I can add it.